lib/rubocop/cop/rails/action_order.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop enforces consistent ordering of the standard Rails RESTful
+      # controller actions.
+      #
+      # The cop is configurable and can enforce any ordering of the standard
+      # actions. All other methods are ignored.
+      #
+      # [source,yaml]
+      # ----
+      #  Rails/ActionOrder:
+      #    ExpectedOrder:
+      #      - index
+      #      - show
+      #      - new
+      #      - edit
+      #      - create
+      #      - update
+      #      - destroy
+      # ----
+      #
+      # @example
+      #   # bad
+      #   def index; end
+      #   def destroy; end
+      #   def show; end
+      #
+      #   # good
+      #   def index; end
+      #   def show; end
+      #   def destroy; end
+      class ActionOrder < Base
+        extend AutoCorrector
+        include VisibilityHelp
+        include DefNode
+
+        MSG = 'Controller actions should be sorted in the following order: %<expected_order>s. ' \
+              'Action `%<current>s` should appear before `%<previous>s`.'
```

Comment:
The message may be long 😅 Can you tweak to the following?

```suggestion
        MSG = 'Action `%<current>s` should appear before `%<previous>s`.'
```